src/applications/vaos/redux/selectors.js

```diff
@@ -53,3 +53,6 @@ export const selectSystemIds = state =>
 
 export const selectFeatureHomepageRefresh = state =>
   toggleValues(state).vaOnlineSchedulingHomepageRefresh;
+
+export const selectFacilitySelectionV22 = state =>
```

Comment:
Could you update this to `selectFeatureFacilitySelectionV22`? That's the naming convention we're trying to use for feature toggle selectors.